api/model/subgraph.go

```diff
@@ -7,5 +7,5 @@ type SubGraphSpec struct {
 }
 
 type SubGraphStatus struct {
-	APIID string `json:"api_id"`
+	LinkedApiDefID string `json:"linked_api_def_id,omitempty"`
```

Comment:
Should we name it as `LinkedByAPI` to follow same naming convention?

Reply:
Thanks @komalsukhani . Updated accordingly in [75a6588](https://github.com/TykTechnologies/tyk-operator/pull/522/commits/75a6588f71da54f612f85a93a85b046c3d89568b)